graph/options/options.go

```diff
@@ -16,8 +16,9 @@ import (
 )
 
 const (
-	AppenderAll  string = "_all_"
-	NamespaceAll string = "all"
+	AppenderAll          string = "_all_"
+	NamespaceAll         string = "all"
+	NamespaceIstioSystem string = "istio-system"
```

Comment:
I was pretty sure that we had it configured somewhere from config.yaml, but it turns out it's wrong. Except in grafana and jaeger config, but that's not appropriate here.

So, the question is not specific to this PR but more to kiali in general, can we not have istio installed elsewhere than in istio-system?

Reply:
I honestly don't know if Istio can show up in a different namespace that istio-system.  I think at least for now this is a fair assumption.